aten/src/ATen/xpu/profiler/XPUProfilerMacros.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <libkineto.h>
+#include <output_base.h>
+#include <time_since_epoch.h>
+
+#include <pti/pti_view.h>
+
+#include <c10/xpu/XPUMacros.h>
+
+namespace at::kineto_plugin::xpu {
+
+#define ptiCheck(returnCode)                                        \
+  if (returnCode != PTI_SUCCESS) {                                  \
+      throw std::runtime_error("PTI tracing got fatal error"        \
+        + " for XPU-based kineto profiler(error code:"              \
+        + std::string(returnCode) + ") Please file issue to XPU");  \
```

Comment:
enum cannot be print as string. PTI should return std exception.

Reply:
I remember, they rejected the exception proposal, right?
What's the next? An new API to map error code to string?

Reply:
They reject the exception for keeping the c-style function.
We ask for a mapping function to translate the error code to const char*.

Reply:
Let's ask them a ETA.

Reply:
ok